tomls/governance/councils-satellite.toml

```
@@ -0,0 +1,38 @@
+[clone.governance_ambassador_council]
+source = "<%= settings.governance_package %>"
+options.salt = "<%= settings.governance_salt %>-ambassador"
```

Instead of salt, when deploying packages, use target preset.

Following up my comment about salt as an example this should become
```suggestion
[clone.governance_ambassador_council]
source = "<%= settings.governance_package %>"
target = "<%= settings.governance_package %>@ambassador" 
options.salt = "<%= settings.salt %>"
```

Salt is more like a global flag to redeploy EVERYTHING and not to make distinction between two packages.

Using different preset will allow this package to be interacted with in cannon UI. Changing salt will not work the same way.

This same comment is applicable for ALL other deployments in this PR

---

fixed in #370 but using `targetPreset` because `target` in current version of cannon does not allow to mix template value and non-template text